infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -514,17 +527,18 @@ private void runHealingDetectorTask() {
                 if (!pollReport.getHealingNodes().isEmpty()) {
                     try {
                         log.info("Attempting to heal nodes in poll report: {}", pollReport);
-                        corfuRuntime.getLayoutView().getRuntimeLayout(layout)
+                        corfuRuntime
+                                .getLayoutView()
+                                .getRuntimeLayout(layout)
                                 .getManagementClient(getLocalEndpoint())
-                                .handleHealing(pollReport.getPollEpoch(),
-                                        pollReport.getHealingNodes())
+                                .handleHealing(pollReport.getPollEpoch(), pollReport.getHealingNodes())
                                 .get();
                         log.info("Healing nodes successful: {}", pollReport);
                     } catch (InterruptedException | ExecutionException e) {
+                        Thread.currentThread().interrupt();
```

Comment:
good practice to set the interrupt flag of the thread

Reply:
Yep, this is exactly what I did